requirements.txt

```diff
@@ -1,6 +1,12 @@
 pip>=1.4
 setuptools>=0.9
 wheel>=0.21
-bumpversion>=0.3
-isodate>=0.5
-requests>=2.4
+bumpversion==0.5.3 \
+    --hash=sha256:6753d9ff3552013e2130f7bc03c1007e24473b4835952679653fb132367bdd57 \
+    --hash=sha256:6744c873dd7aafc24453d8b6a1a0d6d109faf63cd0cd19cb78fd46e74932c77e
+isodate==0.6.0 \
+    --hash=sha256:aa4d33c06640f5352aca96e4b81afd8ab3b47337cc12089822d6f322ac772c81 \
+    --hash=sha256:2e364a3d5759479cdb2d37cce6b9376ea504db2ff90252a2e5b7cc89cc9ff2d8
+requests==2.21.0 \
```

Comment:
Is this because of a conflict with these libraries and versions used in django-api-service?

Hopefully it's ok that we're pinning `requests` here to `2.21.0`, which is disjoint from the `>=2.4` they had.

Reply:
Talked about this offline.